mlflow/store/tracking/rest_store.py

```diff
@@ -218,6 +224,21 @@ def create_trace(self, experiment_id, start_time, end_time, status, attributes,
         response_proto = self._call_endpoint(CreateTrace, req_body)
         return TraceInfo.from_proto(response_proto.trace_info)
 
+    def get_trace_info(self, trace_id):
+        """
+        Get the trace matching the `trace_id`.
+
+        Args:
+            trace_id: String id of the trace to fetch.
+
+        Returns:
+            The fetched Trace object, of type ``mlflow.entities.TraceInfo``.
+        """
+        req_body = message_to_json(GetTraceInfo(trace_id=trace_id))
+        endpoint = f"{_REST_API_PATH_PREFIX}/mlflow/traces/{trace_id}/info"
+        response_proto = self._call_endpoint(GetTraceInfo, req_body, endpoint=endpoint)
```

Comment:
Is it possible to do something like `GetTraceInfo.path.format(trace_id=trace_id)`?

Reply:
It will be a bit complex to pull the path:
```
endpoints = service_method.GetOptions().Extensions[databricks_pb2.rpc].endpoints
```

Maybe we just keep the hardcoded path?

Reply:
Can we create a helper function for that?

Reply:
makes sense, done!